packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -47,7 +47,7 @@
     "@rollup/plugin-node-resolve": "^9.0.0",
     "@rollup/plugin-replace": "^2.4.1",
     "@rollup/plugin-typescript": "^6.0.0",
-    "@rollup/plugin-url": "^5.0.1",
+    "@rollup/plugin-url": "^6.0.0",
```

Comment:
Do we need it?

Reply:
If yes, then it needs a bump and a release

Reply:
Removed